.github/workflows/dockerize-tag.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and Push Tag
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+jobs:
+  Build-Push-Tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: olegtarasov/get-tag@v2.1
+        id: tagName
```

Comment:
Use consistent casing. Since all the other ids are snake case, this one should be too. Recommend `get_tag` instead of tagName.

Reply:
Also, I'd prefer if the Docker tag doesn't contain the letter `v` at the start. So add a tagRegex to strip that out, as shown on the doc page for this step.

Reply:
> Also, I'd prefer if the Docker tag doesn't contain the letter `v` at the start. So add a tagRegex to strip that out, as shown on the doc page for this step.

I am not so good at regex so I might need a bit more help.  Do I add the following:

```
 tagRegex: "v-(.*)"
```

Reply:
I would assume that `^v(\d+\.\d+\.\d+)$` is what you're looking for, which will match a semver-style version number.

Reply:
> I would assume that `^v(\d+\.\d+\.\d+)$` is what you're looking for, which will match a semver-style version number.

Ok. Before that I will try to use https://github.com/marketplace/actions/parse-semver, I think this will make the future tagging much easier. For example, tags like  :1, :1.1, :1.1.1 etc.

Currently, with parse-semver action, it fails to run when I push a tag, may be due to some syntax error in yml or something else.

Reply:
I think you'll need need to put the `get-tag` step first, and use the `$GIT_TAG_NAME` variable you get from that as the sem-ver input string. Because right now it looks like you're trying to extract the semver fields out of a static string, not the actual tag you just tagged with your git client.

Reply:
I think it should be ready now.